### PR TITLE
Document that new pages should be linked to from the global nav

### PR DESCRIPTION
**Pull request by ryan-blunden**

### CODE REVIEW

File: handbook/documentation/index.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ We use the [monthly release blog post](https://about.sourcegraph.com/blog) as a
 
 ## Naming and linking documentation pages
 
-1. There is no global index or nav, so all docs should be linked from their parent index page. Every new document should be cross-linked to its related documentation, and linked from its topic-related index, when it exists.
+1. All docs should be linked to from their parent index page.
```
Comment (nicksnyder) [inline handbook/documentation/index.md]: What exactly do you mean by this? Every page in the current directory should be linked to in the first `index.md` in the directory hierarchy?

File: handbook/documentation/index.md [hunk 2]
```diff
@@ -12,7 +12,10 @@ We use the [monthly release blog post](https://about.sourcegraph.com/blog) as a
...
+1. Every new document should be:
+    - Cross-linked to its related documentation, and linked from its topic-related index, when it exists.
+    - Linked to from the [docs global nav template](https://github.com/sourcegraph/sourcegraph/blob/master/doc/_resources/templates/document.html#L58).
```
Comment (nicksnyder) [inline handbook/documentation/index.md]: It is unclear which parts of this document apply to product documentation, handbook documentation or both. This line item is the former I think. Can we clarify this more?

Comment (ryan-blunden) [reply handbook/documentation/index.md]: I'm addressing this in https://github.com/sourcegraph/about/pull/600#discussion_r383119316

File: handbook/documentation/index.md (hunk 1 above)
Comment (ryan-blunden) [reply handbook/documentation/index.md]: I've changed this to plain language largely inspired by your question.

**Pull request merged**